ratis-client/src/main/java/org/apache/ratis/client/api/SetLeaderElectionApi.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.ratis.client.api;
+
+import org.apache.ratis.protocol.RaftClientReply;
+
+import java.io.IOException;
+
+/**
+ * An API to support control leader election
+ * such as pause and resume election
+ */
+public interface SetLeaderElectionApi {
+
+  /** pause leader election. */
+  RaftClientReply pause(long timeoutMs) throws IOException;
+
+  /** resume leader election. */
+  RaftClientReply resume(long timeoutMs) throws IOException;
```

Comment:
We should not have the timeoutMs parameter since it won't be used in the server.